showcase/scripts/start_otelcol.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+killall otelcol
+
+# Define the directory where you want to install everything
+install_dir="$1"
+
+# Create the install directory if it doesn't exist
+mkdir -p "$install_dir"
+
+# Change directory to the install directory
+cd "$install_dir"
+
+## in future iterations/improvement, make this version dynamic
+curl --proto '=https' --tlsv1.2 -fOL https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.93.0/otelcol_0.93.0_linux_amd64.tar.gz
+tar -xvf otelcol_0.93.0_linux_amd64.tar.gz
```

Comment:
@ddixit14 I've figured out the issue when pulling this down locally. I'm running on my M1 Mac with ARM and this is pulling down linux_amd64.

What worked for me was to pull down `otelcol_0.94.0_darwin_arm64.tar.gz` and run that. Something to consider for the different machines that we're running on. Maybe we can use a docker image for otel collector locally